portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
+	@ignore = "true"
+	@priority = "4"
+	test AssertTotalAmountOfViewsInAllCards {
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		task ("Add a new user in DXP") {
+			JSONUser.addUser(
+				userEmailAddress = "ac@liferay.com",
+				userFirstName = "ac",
+				userLastName = "ac",
+				userScreenName = "ac");
+		}
+
+		task ("Sync Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Login new user and view page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "ac@liferay.com",
+				userLoginFullName = "ac ac");
+		}
+
+		task ("Interation with page") {
+			ACUtils.navigateToSitePageWithUser(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("AC launch") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
```

Comment:
Hey @calvinctvo, I think that this way the test will not know what the value of the `propertyName` variable, we have two cases of how to use the `switchProperty` macro:

Case 1: (more apropriate)
When we don't create the `propertyName` variable the same way you did in your test, we need to use `assignedPropertyName` when we want to access the value of the property name we are using in the test, this happens because `ACDXPSettings.connectDXPtoAnalyticsCloud` macro [returns](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro#L70) [globally to all tests that use it](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro#L130)

Case 2:
When we create the propertyName variable by doing:
`var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");`
That way we can use propertyName to access the variable name